runners/gradle-plugin/build.gradle.kts

```diff
@@ -8,15 +8,14 @@ plugins {
 }
 
 dependencies {
-    api(project(":core"))
-
-    compileOnly("org.jetbrains.kotlin:kotlin-gradle-plugin")
-    compileOnly("com.android.tools.build:gradle:4.0.1")
-    compileOnly(gradleKotlinDsl())
-    testImplementation(project(":test-utils"))
-    testImplementation(gradleKotlinDsl())
-    testImplementation("org.jetbrains.kotlin:kotlin-gradle-plugin")
-    testImplementation("com.android.tools.build:gradle:4.0.1")
+    implementation(projects.core)
```

Comment:
Previously it was declared as `api(..)` - does it make any difference in the case of a Gradle plugin?

Reply:
Good spot, thanks